shared/fs/header/container.js

```diff
@@ -22,7 +25,10 @@ const mapDispatchToProps = (dispatch, {path, routePath}: OwnProps) => ({
         ...Util.tlfToParticipantsOrTeamname(Types.pathToString(path)),
       })
     ),
-  onBack: () => dispatch(putActionIfOnPath(routePath, navigateUp())),
+  onBack:
+    isMobile && path === Constants.defaultPath
+      ? () => dispatch(putActionIfOnPath(routePath, navigateTo([TabsConstants.settingsTab])))
```

Comment:
why not just fix the route so navigateUp works

Reply:
I could figure out a way to make switch to work when user taps “Cancel” in the destination picker (for moveOrCopy), so just went with having folders route and destination picker route as siblings, which means they are both Uber fsRoute and that the fsRoute itself doesn’t render a component. See notes in https://github.com/keybase/client/blob/master/shared/fs/routes.js

Please let me know if there are better route fixes!

Reply:
oops s/could/couldn't